pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Record.java

```diff
@@ -48,7 +48,7 @@
         return Optional.empty();
     }
 
-    default Schema<T> getSchema() {
+    default Schema getSchema() {
```

Comment:
why are we removing this template parameter ?
it is wrong, if the Record is of type T, than the Schema is for type T

Reply:
i will change it.